### PR TITLE
Show unfinalised data entries on polling station choice page

### LINKED ISSUE #369: Show unfinalised data entries on polling station choice page
Show unfinalized data entries for the current user on the select polling station page ([Figma](https://www.figma.com/design/zZlFr8tYiRyp4I26sh6eqp/Kiesraad---Abacus-optelsoftware?node-id=4984-25327&node-type=frame&t=xmRE9rXVKD2bdNY2-11)). This also implies expanding the backend `PollingStationStatus` with an "in progress" state (`first_entry_in_progress`??).

### PR DESCRIPTION
**Pull request by cikzh**
Closes #369

- Adds new status: first_entry_in_progress
- Adds info-level alert to inform the user there are still polling stations in progress
- Adds the ability to add a `ReactElement` to a `Badge`, so we can use the pencil icon
- Removes commented-out Badge variants, as they are being revisited and only add noise
- Adds tests for the icon badge

### CODE REVIEW

**Comment by cikzh** [comment]
@lkleuver @Lionqueen94 I removed the `fill` property in `frontend/lib/ui/Icon/Icon.module.css` instead of overriding it. Is that a correct way of doing it, or will that brake something?

**Comment by Lionqueen94** [comment]
> @lkleuver @Lionqueen94 I removed the `fill` property in `frontend/lib/ui/Icon/Icon.module.css` instead of overriding it. Is that a correct way of doing it, or will that brake something?

I don't see it cause any issues, but maybe @lkleuver or @jorisleker see any broken color setting somewhere?

File: frontend/app/component/form/data_entry/polling_station_choice/PollingStationChoiceForm.tsx [hunk 1]
```diff
@@ -66,13 +67,32 @@ export function PollingStationChoiceForm({ anotherEntry }: PollingStationChoiceF
...
+              Je bent begonnen met het invoeren van onderstaande stembureaus, maar hebt deze nog niet helemaal afgerond:
+            </p>
+            {inProgress.map((pollingStation) => {
+              return pollingStation === undefined ? null : (
+                <PollingStationLink key={pollingStation.id} pollingStation={pollingStation} />
+              );
+            })}
+          </Alert>
+        </div>
+      )}
```
Comment (praseodym) [inline frontend/app/component/form/data_entry/polling_station_choice/PollingStationChoiceForm.tsx:95]: The design has an [x] button to close this alert that's still missing in the implementation.

**Review by praseodym — CHANGES_REQUESTED** [review]
Looks good overall, some small findings:
- The API tests need to be extended to also cover the new `FirstEntryInProgress` status.
- As you mentioned during the previous epic review, the backend code can use some doc comments. Please add some to the functions you edited in this branch.
- And see code comment.

File: frontend/app/component/form/data_entry/polling_station_choice/PollingStationChoiceForm.tsx (hunk 1 above)
Comment (cikzh) [reply frontend/app/component/form/data_entry/polling_station_choice/PollingStationChoiceForm.tsx]: Yes, good catch, I forgot about that one. Fixed in [38c267a](https://github.com/kiesraad/abacus/pull/404/commits/38c267a03e0b3bf0b5759e4c75971685d391552e)

**Review by praseodym — APPROVED** [review]

**Review by Lionqueen94 — APPROVED** [review]

**Pull request merged**